### PR TITLE
ScalaPB: various fix and improves

### PR DESCRIPTION
**Pull request by chikei**
* fix destination path issue mentioned in https://github.com/lihaoyi/mill/pull/1002#issuecomment-737769495
* isolated classloader as #1026
* use T.worker

### CODE REVIEW

File: contrib/scalapblib/test/src/TutorialTests.scala [hunk 1]
```diff
@@ -33,21 +34,19 @@ object TutorialTests extends TestSuite {
 
   object TutorialWithAdditionalArgs extends TutorialBase {
     object core extends TutorialModule {
-      override def scalaPBAdditionalArgs = Seq(
-        s"--zio_out=..."
-      )
+      override def scalaPBAdditionalArgs = T{Seq(
+        "--additional-test=..."
+      )}
```
Comment (lefou) [inline contrib/scalapblib/test/src/TutorialTests.scala:39]: What's the point of changing this?

**Review by lefou — COMMENTED** [review]
Looks good to me. I added one question below.

Also, I just recognized that we have a commented-out test case for ScalaPB. Could you please check if this test works for you locally? I'm pretty sure we can handle non-abilities of the CI env better than commenting out. (Can't tell if there a equivalent to  scalatest's `assume` in utest). I could imagine that this test will work on Github Actions as-is.

**Comment by chikei** [comment]
Ya, I've verified the `calledDirectly` test works locally before push.

File: contrib/scalapblib/test/src/TutorialTests.scala (hunk 1 above)
Comment (chikei) [reply contrib/scalapblib/test/src/TutorialTests.scala:39]: I was written from the version before last scalapb pr and thinking "additional-test" might easier for other people to reasoning and not specific for zio, not something matters in term of code execution.

**Comment by lefou** [comment]
Thank you!

**Pull request merged**